src/imperator/data/ambitions.rs

```diff
@@ -36,7 +36,7 @@ impl DbKind for Ambition {
 
         vd.req_field("can_be_picked");
         vd.req_field("finished_when");
-        vd.req_field("abort");
+        vd.field("abort");
```

Comment:
This line should probably just be removed. You're doing `vd.field_validated_block("abort"` below.